### PR TITLE
feat(j-s): Add appeal case files

### PR DESCRIPTION
**Pull request by unakb**
[Bjóða upp á möguleika að senda inn gögn til Landsréttar utan kæru og greinargerðar](https://app.asana.com/0/1199153462262248/1205724728618364)

## What

Allow prosecutors and defenders to add extra case files to an appeal 

## Why

In case they forgot to add a file and need to do it after completing the appeal brief and statement steps

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/judicial-system/consts/src/lib/consts.ts [hunk 1]
```diff
@@ -102,10 +102,12 @@ export const CREATE_INDICTMENT_ROUTE = '/akaera/ny'
 export const DEFENDER_ROUTE = '/verjandi/krafa'
 export const DEFENDER_INDICTMENT_ROUTE = '/verjandi/akaera'
 export const DEFENDER_APPEAL_ROUTE = '/verjandi/kaera'
+export const DEFENDER_APPEAL_FILES_ROUTE = '/kaera/gogn'
 export const DEFENDER_STATEMENT_ROUTE = '/verjandi/greinargerd'
 export const DEFENDER_CASES_ROUTE = '/verjandi/krofur'
 
 export const APPEAL_ROUTE = '/kaera'
+export const APPEAL_FILES_ROUTE = '/kaera/gogn'
```
Comment (oddsson) [inline libs/judicial-system/consts/src/lib/consts.ts:110]: Why do we have two routes with the same value?

Comment (unakb) [reply libs/judicial-system/consts/src/lib/consts.ts:110]: Fixed!

**Review by oddsson — APPROVED** [review]

**Review by gudjong — CHANGES_REQUESTED** [review]
We should merge appeal case file types

**Review by gudjong — APPROVED** [review]
The change that I suggested needs more discussion so I will let this PR pass as is and open a new issue.

**Pull request merged**